singularity-eos/eos/eos_ideal.hpp

```diff
@@ -198,6 +206,8 @@ class IdealGas : public EosBase<IdealGas> {
       thermalqs::density | thermalqs::specific_internal_energy;
   // optional entropy reference state variables
   Real _EntropyT0, _EntropyRho0;
+  // optional mean atomic mass and number
+  MeanAtomicProperties _AZbar;
 };
 
```

Comment:
Should we have some method that can return Cv as a function of Abar for ideal gases (maybe not for this PR):

rho k T / (mu mp) == gm1 rho cv T

==> cv = k / (gm1 mu mp)

Reply:
:+1: Yes this is a good idea. Will address in a subsequent PR, since I'll need this functionality for the ideal electron EOS.